src/core/sys/posix/sys/statvfs.d

```diff
@@ -86,21 +86,61 @@ version(CRuntime_Glibc) {
 }
 else
 {
-    struct statvfs_t
+    version(NetBSD)
```

Comment:
Better to just create your own whole new `NetBSD` block, rather than stick this in here, ie replace the `else` just above with `else version(NetBSD)`, fill it with everything you need, and leave this last `else` block alone.